friends/src/componenet/FriendList.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Friend from './Friend'
+ const FriendList = props => {
```

Comment:
If you wanted to, you could destructure the props right here. You can do something like this:
```
const FriendList = ({ friends}) => { ... }
```